genref/config.yaml

```diff
@@ -0,0 +1,139 @@
+hiddenMemberFields:
+  - "TypeMeta"
+
+externalPackages:
+  - match: ^k8s\.io/apimachinery/pkg/apis/meta/v1\.Duration$
+    target: https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#Duration
+  - match: ^k8s\.io/apimachinery/pkg/types\.UID$
+    target: https://godoc.org/k8s.io/apimachinery/pkg/types#UID
+  - match: ^k8s\.io/apimachinery/pkg/runtime\.RawExtension$
+    target: https://godoc.org/k8s.io/apimachinery/pkg/runtime/#RawExtension
+
+  - match: ^k8s\.io/apimachinery/pkg/runtime\.Unknown$
+    target: https://godoc.org/k8s.io/apimachinery/pkg/runtime#Unknown
+  - match: ^time\.Duration$
+    target: https://godoc.org/time#Duration
+  - match: ^k8s\.io/(api|apimachinery/pkg/apis)/
+    target: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#{{- lower .TypeIdentifier -}}-{{- arrIndex .PackageSegments -1 -}}-{{- arrIndex .PackageSegments -2 -}}
+
+hideTypePatterns:
+  - "ParseError$"
+  # - "List$"
+
+markdownDisabled: false
+
+stripPrefix:
+  - k8s.io/api/
+  - k8s.io/apimachinery/pkg/apis/
+
+apis:
+  - name: kubelet-config
+    package: k8s.io/kubelet
+    path: config/v1beta1
+    includes:
+      - k8s.io/component-base/config/v1alpha1
+
+  - name: kubelet-stats
+    package: k8s.io/kubelet
+    path: pkg/apis/stats/v1alpha1
+    # This is skipped because the 'stats' API is not yet available in 0.19.3
+    skip: true
+
+  - name: kube-scheduler-policy-config
```

Comment:
Hi @tengqm . I have a question about maintaining this configuration.
How do you know when a component's configuration is outdated?

Reply:
My assumption is that the API versions will not change too much in each k8s releases. After a new release is out, we will have to review this configuration and check if an API version is gone or if a new version is in. Such information usually can be found in the release notes.

Reply:
OK. So if a type has changed will the tool fail to generate that type?

Reply:
The tool is a wrapper of the gengo library, so type change won't be a problem for source parsing or doc generation. The only concern is about huge refactoring, which has to be handled case by case, i.e. change the `path` and/or `includes` configuration.